recipes/autoconf-archive/all/test_package/conanfile.py

```diff
@@ -0,0 +1,24 @@
+from conans import AutoToolsBuildEnvironment, ConanFile, tools
+import os
+import shutil
+
+required_conan_version = ">=1.36.0"
+
+class TestPackageConan(ConanFile):
+    settings = "os"
```

Comment:
Needed because you're doing a test compilation, and the `tools.cross_building` test also needs `settings` to be filled in.
```suggestion
    settings = "os", "arch", "compiler", "build_type"
```